src/components/views/rooms/RoomTile.js

```diff
@@ -88,16 +87,38 @@ module.exports = React.createClass({
     },
 
     componentWillMount: function() {
+        constantTimeDispatcher.register("RoomTile.refresh", this.props.room.roomId, this.onRefresh);
+        constantTimeDispatcher.register("RoomTile.select", this.props.room.roomId, this.onSelect);
```

Comment:
I was expecting to see a `RoomTile.select` dispatched in RoomList's `componentWillMount` to set up the initial selection state, but I don't see where this is happening?